### ISSUE TITLE
News API is not working

### ISSUE THREAD

**Issue by vinitshahdeo** [vinitshahdeo/Corona-India#37]
Please share some alternatives to [News API](https://newsapi.org/). 

A FREE API to fetch the latest news based on topics will be great as NewsAPI.org has paid services now.

**Comment by rajdama** [comment]
I can do it

**Comment by Shylajhaa** [comment]
@vinitshahdeo Would be interested in working on this issue or some other issue in this project as well

**Comment by saif-malik-01** [comment]
@vinitshahdeo  we can make our own api for that by web scrapping, I make one for live cricket scores.

**Comment by prkhrv** [comment]
@saif-malik-01  Agreed ! We can make our own API by scraping from Google News etc etc

**Comment by lalit-commits** [comment]
can I fix it ?
